spec.html

```diff
@@ -26831,7 +26831,10 @@ <h1>Date ( _year_, _month_ [ , _date_ [ , _hours_ [ , _minutes_ [ , _seconds_ [
             1. If _minutes_ is present, let _min_ be ? ToNumber(_minutes_); else let _min_ be 0.
             1. If _seconds_ is present, let _s_ be ? ToNumber(_seconds_); else let _s_ be 0.
             1. If _ms_ is present, let _milli_ be ? ToNumber(_ms_); else let _milli_ be 0.
-            1. If _y_ is not *NaN* and 0 &le; ToInteger(_y_) &le; 99, let _yr_ be 1900+ToInteger(_y_); otherwise, let _yr_ be _y_.
+            1. If _y_ is *NaN*, let _yr_ be *NaN*.
+            1. Else,
+              1. Let _yi_ be ! ToInteger(_y_).
+              1. If 0 &le; _yi_ &le; 99, let _yr_ be 1900+_yi_; otherwise, let _yr_ be _y_.
```

Comment:
47cace07e7b646a21c83b6dceda6b84d6d6afca7: let's add spaces around `1900`, `+`, and `_yi_`

also, "otherwise" seems odd - should this be "else"?

Reply:
>  let's add spaces around 1900, +, and _yi_

That's PR #1162.

> also, "otherwise" seems odd - should this be "else"?

I just used the same word as in the prior version. But for a one-line 'if' command, "otherwise" is actually more popular than "else" (97 vs 80 occurrences).

If the editors have a preference, I could add a commit to consistify the syntax, but maybe that should be a separate PR.